File: Java/src/main/java/com/funcionariosveiculos/Domain/Models/Funcionario.java
package com.funcionariosveiculos.Domain.Models;

import java.time.LocalDateTime;
import java.time.ZonedDateTime;

public class Funcionario {

    public Integer Id;
    public String Nome;
    public LocalDateTime DataCadastro;

    public Funcionario(String nome) {
        this.Id = 0;
        this.Nome = nome;
        this.DataCadastro = ZonedDateTime.now().toLocalDateTime();;
    }

    public Funcionario(Integer id, String nome, LocalDateTime dataCadastro) {
        this.Id = id;
        this.Nome = nome;
        this.DataCadastro = dataCadastro;
    }

    @Override
    public String toString() {
        return "Funcionário - " + Id + " - " + Nome;
    }
}

File: Python/Domain/Models/Funcionario.py
import datetime

class Funcionario:
    def __init__(self, nome, id = 0, dataCadastro = datetime.datetime.now()):
        self.__id = id
        self.__nome = nome
        self.__dataCadastro = dataCadastro

    @property
    def nome(self):
        return self.__nome
    
    @property
    def id(self):
        return self.__id
    
    @property
    def dataCadastro(self):
        return self.__dataCadastro
    
    def __str__(self):
        return 'Funcionário - ' + self.__id.__str__() + ' - ' + self.__nome